packages/@react-aria/toolbar/docs/useToolbar.mdx

```diff
@@ -0,0 +1,170 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/toolbar';
+import typesDocs from 'docs:@react-types/shared/src/events.d.ts';
+import {HeaderInfo, FunctionAPI, TypeContext, TypeLink, InterfaceType, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-aria/toolbar/package.json';
+import Anatomy from './toolbar-anatomy.svg';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+import tailwindPreview from 'url:./tailwind.png';
+
+---
+category: Content
+keywords: [toolbar, aria]
+---
+
+# useToolbar
+
+<PageDescription>{docs.exports.useToolbar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useToolbar']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useToolbar} links={docs.links} />
+
+## Features
+
+There is no native element to implement a toolbar in HTML without javascript. `useToolbar`
+helps achieve accessible toolbar components that can be styled as needed.
+
+* Exposed to assistive technology as a `toolbar` using ARIA
+* Support for keyboard navigation
+* Support for both horizontal and vertical orientations
+* Support for react aria based children, button, togglebutton, menu, checkbox, and link
```

Comment:
Nit:
```suggestion
* Support for React Aria based children, button, togglebutton, menu, checkbox, and link
```
also is this supposed to be a list of available elements buildable via our hooks or a just a list of supported elements? (aka should this be "Support for React Aria based children: button, togglebutton, menu, checkbox, and link"?)

Reply:
it's somewhat non-exhaustive list of supported components, but also meant to indicate not everything is supported
for instance, I know that Select doesn't work right now